fbw-a32nx/src/wasm/systems/a320_systems/src/hydraulic/sfcc/flaps_channel.rs

```diff
@@ -9,8 +9,9 @@ use systems::simulation::{
 };
 
 use uom::si::{angle::degree, f64::*, velocity::knot};
+use uom::ConstZero;
 
-use super::sfcc::SlatFlapControlComputerMisc;
+use super::SlatFlapControlComputerMisc;
 
 pub struct FlapsChannel {
```

Comment:
```suggestion
pub(super) struct FlapsChannel {
```
Same with the functions.